lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -125,30 +129,15 @@ std::string _GenerateXMLString(const HdMaterialNetwork& materialNetwork, bool in
     std::string result;
 
     if (ARCH_LIKELY(!materialNetwork.nodes.empty())) {
-        const size_t numNodes = materialNetwork.nodes.size();
-
-        SdfPathVector paths;
-        paths.reserve(numNodes);
-
-        for (size_t i = 0; i < numNodes; i++) {
-            paths.push_back(materialNetwork.nodes[i].path);
-        }
-
-        // We use concise relative paths to generate the same string for duplicate material
-        // networks.
-        const SdfPathVector conciseRelativePaths = SdfPath::GetConciseRelativePaths(paths);
```

Comment:
Now that we've created the simplified paths which have only one unique token, we don't need to worry about name collision inside a material network so we can remove the part of the code here.